sql/src/test/java/io/crate/integrationtests/RenameTableIntegrationTest.java

```diff
@@ -69,13 +69,13 @@ public void testRenameTableEnsureOldTableNameCanBeUsed() {
 
     @Test
     public void testRenameClosedTable() {
-        execute("create table t1 (id int) with (number_of_replicas = 0)");
-        execute("insert into t1 (id) values (1), (2)");
+        execute("create table foo.t1 (id int) with (number_of_replicas = 0)");
```

Comment:
We don't have a test for the default schema anymore. Can you please add one for that as well?